web/controllers/user_controller.ex

```diff
@@ -62,4 +72,11 @@ defmodule Pairmotron.UserController do
     |> put_flash(:info, "User deleted successfully.")
     |> redirect(to: user_path(conn, :index))
   end
+
+  defp current_assigned_user_id(conn) do
```

Comment:
Near duplicate of https://github.com/ElixirCLE/pairmotron/pull/31/files#diff-c9ec1fc6e91a7bd967c8f8506027d1f9R27.

See the note for https://github.com/ElixirCLE/pairmotron/pull/31/files#diff-7dc24a26994057cb34cfbdb463c15b3fR44